src/engine/framework/BaseCommands.cpp

```diff
@@ -795,12 +795,9 @@ namespace Cmd {
 
     struct aliasRecord_t {
         std::string command;
-        int lastRun;
```

Comment:
The "intent" was that the author imagined that commands would be executed recursively instead of using a buffer (see my commit message for more info). The command system is Turing-complete in various ways so no avoiding infinite loops really